docs/HowToUseTaskMonitor.md

```diff
@@ -0,0 +1,79 @@
+# How to use Task-monitor
+
+Task monitor is kernel thread which monitor the tasks/pthreads.
+It checks if registered tasks/pthreads are alive.
+
+## Contents
+- [Enable task-monitor](#enable-task-monitor)
+- [Use task-monitor](#use-task-monitor)
+
+
+## Enable task-monitor
+To enable task-monitor, please find four steps as shown below:
+
+## 1. Enable task monitor
```

Comment:
How about making this to sub-category of `Enable task-monitor` with `###`?

Reply:
modified